From e7c167c84a7bb9c04dd802d651c6e3911c5ffef5 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Tue, 01 Oct 2024 10:56:01 +0300
Subject: [PATCH] [SW-0] image: fix preboot string print

Until 3 weeks ago, the ppboot printed the following string
"Preboot loadaddr:". Patch number a4a9506 refactored the code
to use the function 'name' parameter for the print, which caused
it to show as "PREBOOT loadaddr:". this confused some CI jobs like
the Coral CI.
This patch restores the print to it's previous form by adjusting
the preboot image name passed to ppboot_cp_image().

Change-Id: I0898cbfc98df95d686078066f6455c4bffb387ce
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/app/ppboot/ppboot.c b/app/ppboot/ppboot.c
index b407c94..4664fc6 100644
--- a/app/ppboot/ppboot.c
+++ b/app/ppboot/ppboot.c
@@ -51,7 +51,7 @@
 	preboot_puts_hex_raw(", size: ", desc->img_size, false);
 	preboot_puts(")\n");
 
-	is_valid = ppboot_cp_image(desc->loadaddr, desc->spi_offset, desc->img_size, "PREBOOT");
+	is_valid = ppboot_cp_image(desc->loadaddr, desc->spi_offset, desc->img_size, "Preboot");
 	if (is_valid && desc->desc_ver > 1) {
 		is_valid = ppboot_is_valid_img_desc(desc);
 		if (!is_valid)
